modules/mobile/config/initializers/statsd.rb

```diff
@@ -88,17 +100,6 @@
 StatsD.increment('mobile.profile.link_account.success', 0)
 StatsD.increment('mobile.profile.link_account.failure', 0)
```

Comment:
Having a hard time figuring out why this file has so many changes. It looks the same. 🤔

Reply:
I was taking lines out of the `Rails.application.reloader.to_prepare do` block and then putting them back in trying to figure out what needed to be in there and what didn't, so I probably ended up getting things out of the order they were originally in.

Reply:
Why not just put them all in there?

Reply:
We certainly could I guess. I'll make that change and push it up.